ios/wrappers/push/NativeMessagingPush.swift

```diff
@@ -12,27 +12,35 @@ enum PushPermissionStatus: String, CaseIterable {
     }
 }
 
-@objc(CioRctPushMessaging)
-class CioRctPushMessaging: NSObject {
-    @objc static func requiresMainQueueSetup() -> Bool {
-        false /// false because our native module's initialization does not require access to UIKit
+@objc(NativeMessagingPush)
+public class NativeMessagingPush: NSObject {
+    private let logger: CioInternalCommon.Logger = DIGraphShared.shared.logger
+
+    @objc
+    public func onMessageReceived(
+        _ message: NSDictionary,
+        handleNotificationTrigger: Bool,
+        resolve: @escaping RCTPromiseResolveBlock,
+        reject: @escaping RCTPromiseRejectBlock
+    ) {
+        Preconditions.unsupportedOnIOS(methodName: "onMessageReceived")
```

Comment:
We're already suppressing this in TS files, but need to override due to Turbo spec generation